src/main/java/com/amazon/opendistroforelasticsearch/security/securityconf/impl/WhitelistingSettings.java

```diff
@@ -0,0 +1,53 @@
+package com.amazon.opendistroforelasticsearch.security.securityconf.impl;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class WhitelistingSettings {
+    @JsonProperty(value = "whitelistingEnabled")
+    private boolean whitelistingEnabled;
+    @JsonProperty(value = "whitelistedAPIs")
+    private List<String> whitelistedAPIs;
+
+    public WhitelistingSettings() {
+        whitelistingEnabled = false;
+        whitelistedAPIs = new ArrayList<>(Arrays.asList(
```

Comment:
Should this be an empty list? Why are these enabled by default?

Reply:
Since whitelistingEnabled is false by default, this list doesn't really matter, so I didn't change it while making the PR. I can see how this is confusing though, so I'll make it an empty list so that it makes more sense

Reply:
Initially, I had whitelisted these APIs because they were required for some cluster tests when I was making the standalone plugin, and I forgot to change this. Didn't need to change it because whitelistedAPIs is read from the index as soon as the cluster starts, so it is irrelevant. For clarity, I have made this change though.